nanopub/publication.py

```diff
@@ -8,6 +7,9 @@
 from nanopub import namespaces, profile
 from nanopub.definitions import DUMMY_NANOPUB_URI
 
+# To be replaced with the published uri upon publishing
+DUMMY_NAMESPACE = rdflib.Namespace(DUMMY_NANOPUB_URI + '#')
```

Comment:
I realised this is actually constant now. Makes it much cleaner!

Reply:
👍